packages/runtime/test/lib/AdminApiClient.ts

```diff
@@ -45,5 +45,5 @@ export async function cancelIdentityDeletionProcessFromBackboneAdminApi(
 
     await syncUntilHasIdentityDeletionProcess(transportService, identityDeletionProcessId);
 
-    return await transportService.identityDeletionProcesses.getIdentityDeletionProcess({ id: identityDeletionProcessId });
+    return transportService.identityDeletionProcesses.getIdentityDeletionProcess({ id: identityDeletionProcessId });
```

Comment:
actually we want promises to fail where they were called and not where they are returned. Please re-add this await.

I don't know why our eslint rules aren't shouting.